sdk/elements/03-hydra-cli.md

```diff
@@ -316,4 +316,81 @@ Host: :host
+
+this command to creates a monitoring project in your source directory, which will be named `metagraph-monitoring-service`
+
+To use this feature, we need to know the informations about the remote host that will be used as monitoring.
+So, you need to populate the file `infra/ansible/remote/hosts.ansible.yml` under the monitoring section.
+You should provide a user that has sudo privileges without requiring a password. Refer to [this document](https://gcore.com/learning/how-to-disable-password-for-sudo-command/) to learn how to enable password-less sudo for a user.
+
+
+### Monitoring Configuration
+
+Before deploying to remote instances, you need to configure your monitoring by editing the file `config/config.json`. When you run the install command, some fields will be auto-populated based on the `euclid.json` file, which includes:
+
+-   `metagraph.id`: The unique identifier for your metagraph.
+-   `metagraph.name`: The name of your metagraph.
+-   `metagraph.version`: The version of your metagraph.
+-   `metagraph.default_restart_conditions`: Specifies conditions under which your metagraph should restart. These conditions are located in `src/jobs/restart/conditions`, including:
+  -   `SnapshotStopped`: Triggers if your metagraph stops producing snapshots.
+  -   `UnhealthyNodes`: Triggers if your metagraph nodes become unhealthy.
+-   `metagraph.layers`:
+  -   `ignore_layer`: Set to `true` to disable a specific layer.
+  -   `ports`: Specifies public, P2P, and CLI ports.
+  -   `additional_env_variables`: Lists additional environment variables needed upon restart, formatted as `["TEST=MY_VARIABLE, TEST_2=MY_VARIABLE_2"]`.
+  -   `seedlist`: Provides information about the layer seedlist, e.g., `{ base_url: ":your_url", file_name: ":your_file_name"}`.
+-   `metagraph.nodes`:
+  -   `ip`: IP address of the node.
+  -   `username`: Username for SSH access.
+  -   `privateKeyPath`: Path to the private SSH key, relative to the service's root directory. Example: `config/your_key_file.pem`.
+  -   `key_file`: Details of the `.p12` key file used for node startup, including `name`, `alias`, and `password`.
+-   `network.name`: The network your metagraph is part of, such as `integrationnet` or `mainnet`.
+-   `network.nodes`: Information about the GL0s nodes.
+-   `check_healthy_interval_in_minutes`: The interval, in minutes, for running the health check.
+
+NOTE: You must provide your SSH key file that has access to each node. It is recommended to place this under the `config` directory. Ensure that this file has access to the node and that the user you've provided also has sudo privileges without a password.
+
+### Customize Monitoring
+
+Learn how to customize your monitoring by checking the repositories:
+
+-   [metagraph-monitoring-service-package](https://github.com/Constellation-Labs/metagraph-monitoring-service-package)
+-   [metagraph-monitoring-service](https://github.com/Constellation-Labs/metagraph-monitoring-service-package)
```

Comment:
What's the difference between these two repositories? I feel like these should be described separately.